refactor/src/Cage.py

```diff
@@ -132,10 +135,19 @@ def update(self, cur_date: dt.datetime, step_size: int, pressure: int) -> tuple:
         # Lice coming from reservoir
         lice_from_reservoir = self.get_reservoir_lice(pressure)
 
-        if cur_date >= self.start_date:
-            # Events that only happen when cage is populated with fish
-            # (after start date)
+        if cur_date < self.start_date:
+            # Values that are not used before the start date
+            treatment_mortality = {}
+            fish_deaths_natural, fish_deaths_from_lice = 0, 0  # type: Tuple[int, int]
```

Comment:
Why are we using these type annotations? I believe both mypy and pytype can easily tell the type of fish_deaths_natural and fish_deaths_from_lice. Unless it's related to int vs np.int64? (See below)